### PR TITLE
Add first cut of whatsapp type

### PR DESCRIPTION
**Pull request by nicpottier**
Only visibile for beta peeps, this is a first cut, not finished but want to get it merged so peeps can play with it

### CODE REVIEW

File: temba/channels/handlers.py [hunk 1]
```diff
@@ -106,6 +106,18 @@ def post(self):  # pragma: no cover
         return HttpResponse("DMark handling only implemented in Courier.", status_code=401)
 
 
+class WhatsApp(BaseChannelHandler):
```
Comment (rowanseymour) [inline temba/channels/handlers.py]: At some point might be worth having a "CourierOnlyHandler" base class

File: temba/channels/models.py [hunk 2]
```diff
@@ -358,7 +359,7 @@ def create(cls, org, user, country, channel_type, name=None, address=None, confi
         if not schemes:
             raise ValueError("Cannot create channel without schemes")
 
-        if country and schemes != ['tel']:
+        if country and schemes[0] not in ['tel', 'whatsapp']:
```
Comment (rowanseymour) [inline temba/channels/models.py]: Confused... why would a whatsapp channel be country specific?

File: temba/settings_common.py [hunk 3]
```diff
@@ -994,10 +994,9 @@
 CELERY_ACCEPT_CONTENT = ['json']
 CELERY_TASK_SERIALIZER = 'json'
 
-# TODO: uncomment to use SuperAutoscaler
-# CELERYD_AUTOSCALER='temba.celery_autoscaler:SuperAutoscaler'
-AUTOSCALE_MAX_CPU_USAGE = 75
-AUTOSCALE_MAX_USED_MEMORY = 75
+CELERYD_AUTOSCALER = 'temba.celery_autoscaler:SuperAutoscaler'
```
Comment (rowanseymour) [inline temba/settings_common.py]: This intentional?

File: temba/channels/models.py (hunk 2 above)
Comment (nicpottier) [reply temba/channels/models.py]: Channels aren't really "country specific", but their numbers are associated with numbers. WhatsApp numbers are indeed phone numbers so this seemed appropriate to me. You can still send to numbers that aren't in your country but we have a field for country and we know the originating country of a whatsapp number so seemed more right than ignoring it.

File: temba/channels/handlers.py (hunk 1 above)
Comment (nicpottier) [reply temba/channels/handlers.py]: Ya, that's not a bad idea, we still want the reverse URLs so we'd still subclass it but implementation could be  the same.

**Pull request merged**